explainer.md

```diff
@@ -457,14 +708,41 @@ Many thanks for valuable feedback, advice, and tools from:
 Bogdan Brinza and Cynthia Shelley of Microsoft were credited as authors of an
 earlier draft of this spec but are no longer actively participating.
 
-## Appendix: `accessibleNode` naming
+## Appendix: `AccessibleNode` naming
+
+We have chosen the name `AccessibleNode` for the class representing one
+node in the accessibility tree as exposed by the Accessibility Object Model,
+and `accessibleNode` as the method to access the accessible node from a
+DOM element.
+
+In choosing this name, we have tried to pick a balance between brevity,
+clarity, and generality.
+
+* Brevity: The name should be as short as possible, but without
```

Comment:
I would pull the second half of this into "clarity" -

* Brevity: The name should be as short as possible.
* Clarity: The name should reflect the function of the API, without using opaque abbreviations or contractions - the purpose of the property on `Element` should be possible to guess if a developer encounters it while debugging.
* Generality: The name should not be too narrow and limit the scope of the spec.